js/interface/add-markers.js

```diff
@@ -320,16 +320,31 @@ Fliplet.InteractiveMap.component('add-markers', {
       this.allMarkerStyles.push(newItem)
       this.saveData()
     },
-    deleteMarkerStyle(index) {
+    deleteMarkerStyle(marker) {
+      var index = marker.index
```

Comment:
```suggestion
      option = options || {}

      var index = option.index
```

@MaksymShokin It is also good practice to normalise parameters so the code can fail silently.